crates/testenv/Cargo.toml

```diff
@@ -17,9 +17,16 @@ workspace = true
 
 [dependencies]
 bdk_chain = { path = "../chain", version = "0.20.0", default-features = false }
-electrsd = { version = "0.28.0", features = [ "bitcoind_25_0", "esplora_a33e97e1", "legacy" ] }
+electrsd = { version = "0.28.0", features = [ "legacy" ], default-features = false }
+
+[dev-dependencies]
+bdk_testenv = { path = "." }
```

Comment:
hmm, I actually don't know what it does, and tests run fine without it. I just left it in since it was in @ValuedMammal 's suggested changes and didn't hurt anything. appearently it's a 'common pattern' but ya idk what purpose it serves. dev-dependencies are only for tests right?

Reply:
The rationale was that it effectively enables default features for the single test in testenv `test_reorg_is_detected_in_electrsd`, but it's possible we won't notice a difference in CI where we test the whole workspace

Reply:
Oh, got it. I was testing the whole workspace locally and noticed it wasn't required, that's why I asked.

Reply:
ah, interesting!